package.json

```diff
@@ -46,13 +46,15 @@
     "babel": "^6.5.2",
     "browser-headers": "^0.4.1",
     "chai": "^3.5.0",
+    "grpc": "^1.22.2",
```

Comment:
Does this package need to be in "dependencies" since it is imported in the generated `_grpc_pb.d.ts` files, or is it ok here in "devDependencies"?

Reply:
Yep should be fine, I see you are only consuming the `grpc` package in your tests (hence a dev dependency) so shouldn't cause an issue for consumers.